src/v2/Apps/Purchase/Components/PurchaseHistory.tsx

```diff
@@ -26,85 +33,236 @@ const StyledImage = styled(Image)`
   height: 50px;
   width: 50px;
 `
+const getIcon = status => {
+  switch (status) {
+    case "submitted":
+      return <PendingCircleIcon fill="black60" />
+    case "processing":
+      return <PendingCircleIcon fill="black60" />
+    case "canceled":
+      return <XCircleIcon fill="red100" />
+    case "refunded":
+      return <XCircleIcon fill="red100" />
+    case "confirmed":
+      return <CheckCircleFillIcon />
+    case "fulfilled":
+      return <CheckCircleFillIcon />
+  }
+}
+
+const getColor = status => {
+  switch (status) {
+    case "submitted":
+      return "black60"
+    case "processing":
+      return "black60"
+    case "canceled":
+      return "red100"
+    case "refunded":
+      return "red100"
+    case "confirmed":
+      return "black100"
+    case "fulfilled":
+      return "black100"
+  }
+}
 
 const OrderRow = (props: OrderRowProps) => {
   const { order } = props
   const artwork = get(order, o => o.lineItems.edges[0].node.artwork)
+  const { partner } = artwork
 
-  const orderCreatedAt = DateTime.fromISO(order.createdAt).toLocaleString(
-    DateTime.DATE_SHORT
-  )
-  if (!artwork) {
+  const orderCreatedAt = DateTime.fromISO(order.createdAt)
+  if (!artwork || order.state === "ABANDONED") {
     return null
   }
 
+  const { creditCard, requestedFulfillment } = order
+
   const orderIsInactive =
-    order.state === "ABANDONED" || order.state === "CANCELED"
+    order.state === "CANCELED" || order.state === "REFUNDED"
 
+  const partnerImageUrl = get(partner, p => p.profile.icon.url)
+  const partnerInitials = get(partner, p => p.initials)
+  const creditCardNumber = get(creditCard, c => c.lastDigits)
+  const fulfillment = get(requestedFulfillment, r => r.__typename)
```

Comment:
We no longer need `get` now that we support [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)